lnbits/wallets/clnrest.py

```diff
@@ -121,7 +121,7 @@ async def status(self) -> StatusResponse:
         if not data.get("channels"):
             return StatusResponse("no data or no channels available", 0)
 
-        total_our_amount_msat = sum(parse_msat(channel["our_amount_msat"]) for channel in data["channels"])
+        total_our_amount_msat = sum(channel["our_amount_msat"] for channel in data["channels"])
```

Comment:
This breaks if `channel["our_amount_msat"]` does not contain an `int`. Is it guaranteed that it will always contain an `int`?